LICENSE

```diff
@@ -1,21 +1,202 @@
-MIT License
-
-Copyright (c) 2021 Maas Lalani
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+
+                                 Apache License
```

Comment:
I'm curious why the license changed, did you use the cobra generator? I would like to keep MIT if possible.

Reply:
Apache license states: 

>  (a) You must give any other recipients of the Work or
          Derivative Works a copy of this License; 

so I kept this license. TBF don't know how to resolve it.

Reply:
Hmmm, I really doubt that using cobra (as a distributed package) requires you to change your license, (forking and changing it definitely would). For example, [GitHub's cli uses cobra](https://github.com/cli/cli/blob/5a2ec54685806a6576bdc185751afc09aba44408/cmd/gh/main.go#L31) and [is licensed under MIT](https://github.com/cli/cli/blob/trunk/LICENSE)